### ISSUE TITLE
Feature request: Warning if not enough contrast ratio

### ISSUE THREAD

**Issue by smhmd** [tailwindlabs/tailwindcss-intellisense#182]
Optionally (enable/disable in settings) calculate contrast ratio and warn if it's below `4.5:1` unless `text-xl` or higher is also in use in which the minimum becomes `3`, if I'm not mistaken ([spec 1](https://www.w3.org/TR/WCAG21/#contrast-minimum) - [spec 2](https://www.w3.org/TR/WCAG21/#dfn-large-scale).)
```html
<div class="͇b͇g͇-͇g͇r͇a͇y͇-͇8͇0͇0͇ ͇t͇e͇x͇t͇-͇g͇r͇a͇y͇-͇5͇0͇0͇" />
```

**Comment by bradlc** [comment]
Hey @smhmd. This is a cool idea! Unfortunately I think it would be very tricky to implement in a way that avoids false-positives. For example there are many situations where the extension cannot know the elements text size. Even in this basic example the extension does not currently know that the inner element has a font size of `3rem`:

```html
<div class="text-5xl">
  <div class="bg-gray-800 text-gray-500"></div>
</div>
```

The font-size could also be being set in a completely different HTML or CSS file. If we just look at the background and text color classes we risk showing a warning when actually the contrast is fine because the font size is massive.

For that reason I am going to close this, but I appreciate the suggestion! ❤️
